app/redux/wallet/reducer.js

```diff
@@ -86,6 +71,22 @@ const reducer = (state: StoreStateType = initialState, action: Action) => {
       const { transactions } = action.payload;
       return { ...state, transactions };
     }
+    case SET_LAST_USED_ADDRESSES: {
+      const { transactions } = action.payload;
+      const getFirstUniqueTransactions = (txList: TxList, n: number = 3): Contact[] => {
```

Comment:
return value is subset of tx list and not Contact[]

Reply:
mapped to match the props.